napari/_qt/qt_main_window.py

```diff
@@ -757,8 +759,8 @@ def add_dock_widget(
         allowed_areas: Optional[Sequence[str]] = None,
         shortcut=_sentinel,
         add_vertical_stretch=True,
-        menu=None,
         tabify: bool = False,
+        menu: Optional[Union[QModelMenu, NapariMenu]] = None,
```

Comment:
Was nitpick but amended order as `_add_viewer_dock_widget` is always used with `menu` last.